app/views/components/fields/removable_elements_table.scala.html

```diff
@@ -33,17 +33,17 @@
           @title.map { heading =>
               <thead>
                   <tr>
-                    <th id="@{id}-heading" colspan="2">@heading</th>
+                    <th id="@{id}-heading" colspan="2" scope="col">@heading</th>
                   </tr>
               </thead>
           }
           <tbody>
             @for((elem, index) <- elements.zipWithIndex) {
                 <tr id="@{id}__row@index">
-                    <th id="@{id}-row@index-label">@{elem.label}</th>
-                    <th id="@{id}-row@index-remove_button">
+                    <td scope="row" id="@{id}-row@index-label">@{elem.label}</td>
```

Comment:
td ?

Reply:
**TH** is used for table header cells while **TD** is used for table data cells. This distinction gives user agents a means to render such cells distinctly, for instance by using a larger or heavier font for header cells.